docs/start/getting-started/fragments/flutter/integrate.md

```diff
@@ -1,5 +1,7 @@
 In this tutorial, you will integrate basic functionality for **Analytics**.
 
+<amplify-callout> The code won't compile until after you have finished <b>*'Connect to the cloud'*</b> section of the tutorial. </amplify-callout> 
```

Comment:
in order to render markdown correctly, `amplify-callout` requires you to skip a line after the opening tag and before closing it. Also, plain markdown is preferred over `<b>`.

That being said, I think a better fit for it is to actually link to the section of the tutorial you mentioned, here's my suggestion.

```suggestion
<amplify-callout>

The code won't compile until after you have finished [Connect to the cloud](~/start/getting-started/integrate.md) section of the tutorial.

</amplify-callout> 
```